src/modrover/strategies/__init__.py

```diff
@@ -0,0 +1,9 @@
+from modrover.strategies.down_explore import DownExplore
+from modrover.strategies.full_explore import FullExplore
+from modrover.strategies.up_explore import UpExplore
+
+strategy_type_dict = {
```

Comment:
I don't remember if we decided on better strategy names? Forward/backward or add/subtract, etc.?

Reply:
I vote for `forward` and `backward`

Reply:
+1 for forward/backward. Forward means start from 0, add 1 every layer